.github/workflows/tag-and-build.yml

```
@@ -0,0 +1,70 @@
+# This workflow will build the CodeFlare Operator image and push it to the project-codeflare image registry
+
+name: Tag and Release
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Tag to be used for operator image'
+        required: true
+        default: '0.0.0-dev'
+      replaces:
+        description: 'The previous semantic version that this tag replaces.'
+        required: true
+        default: '0.0.0-dev'
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Activate cache
+      uses: actions/cache@v2
+      with:
+        path: /cache
+        key: ${{ runner.os }}-cache-${{ hashFiles('**/go.sum', '.pre-commit-config.yaml') }}
+
+    - name: Create tag
+      uses: actions/github-script@v5
```

```suggestion
      uses: actions/github-script@v6
```
A nitpick: v5 is using deprecated Node 12.